src/renderer/App.tsx

```diff
@@ -47,9 +47,10 @@ export default function App() {
   useEffect(() => {
     async function getSavedExperimentId() {
       const connectionWithoutDots = connection.replace(/\./g, '-');
-      const experimentId = await window.storage.get(
-        `experimentId.${connectionWithoutDots}`
-      );
+      // window.storage should be defined by cloud or electron preload script
+      const experimentId = window.storage
+        ? await window.storage.get(`experimentId.${connectionWithoutDots}`)
+        : 1;
```

Comment:
Does setting it to "1" break if there are no experiments? Shouldn't you set it to '' or null